src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -214,6 +208,28 @@ protected function writePrompt(OutputInterface $output, Question $question)
         $output->write($message);
     }
 
+    /**
+     * @param string $tag
+     *
+     * @return string[]
+     */
+    protected function prepareChoices($tag, ChoiceQuestion $question)
+    {
+        $choices = $question->getChoices();
+
+        $maxWidth = max(array_map([$this, 'strlen'], array_keys($choices)));
+
+        $messages = [];
+
+        foreach ($choices as $key => $value) {
+            $width = $maxWidth - $this->strlen($key);
+            // We using strlen + str_repeat to fix sprintf whitespace padding problem with UTF-8)
+            $messages[] = sprintf('  [<%s>%s%s</%s>] %s', $tag, $key, str_repeat(' ', $width), $tag, $value);
```

Comment:
@nicolas-grekas  `sprintf` left justify does not work with multibytes. Shouldn't we create a `sprintf` method in the abstract Helper class like we did for strlen, substr, etc.? Or using str_repeat for all strings is ok?

Reply:
So, you want to implement sprintf in pure PHP.
Yes, sure... in theory :)
in practice, I don't know :)

Reply:
I mean add a sprintf helper method to handle the left justify. If the string is not utf-8 we use the normal sprintf. If it is, we use the str_repeat strategy.

Reply:
OK, sure!

Reply:
@proggga Would you have time to take of this?

Reply:
Hi, thanks for reviewing this,
Yea, Maybe I can, but I'm not true symphonist, so I don't know rules for creating new helper/naming/best place for it

Reply:
> @proggga Would you have time to take care of this?

May be you can give instructions, and after may be I can do it

Reply:
In your proposed changes, you replaces the sprintf left justify feat by strlen + str_repeat everytime. Firstly, this alternative should only be used for multibytes strings. Secondly, it should be done in a new helper method in the abstract Helper.php class (like we did for strlen and sbustr).